app/README.md

```diff
@@ -20,8 +20,16 @@ app
 |   └── mock/           Data representing mock API responses.
 ├── site/               Data or functions to access site-wide, such as metrics and questions metadata.
 ├── styles/             CSS stylesheets.
+├── tests/e2e           Cypress file structure
```

Comment:
Thanks for adding this! You can change the description from "Cypress file structure" to "End-to-end tests."